plugins/woocommerce/includes/admin/class-wc-admin-notices.php

```diff
@@ -215,8 +227,12 @@ public static function add_notice( $name, $force_save = false ) {
 	 * @param bool   $force_save Force saving inside this method instead of at the 'shutdown'.
 	 */
 	public static function remove_notice( $name, $force_save = false ) {
+		// Don't proceed if the notice doesn't exist.
+		if ( ! self::has_notice( $name ) ) {
```

Comment:
Do you think we should check `$force_save` and run the function if it was set? I could see someone using `remove_notice()` with the expectation that it forces a save rather than calling `store_notices()`. It's not super likely but "force" seems like something you'd be expecting it to do.